data/rubygems.yml

```diff
@@ -10,8 +10,10 @@ activerecord: rails/rails
 activesupport: rails/rails
 capistrano: capistrano/capistrano
 coffee-script-source: jashkenas/coffeescript
+ffi: ffi/ffi
 mixlib-archive: chef/mixlib-archive
 mixlib-authentication: chef/mixlib-authentication
+mixlib-cli: chef/mixlib-cli
```

Comment:
Also, this change will not be necessary when https://github.com/chef/mixlib-cli/blob/v2.0.6/mixlib-cli.gemspec#L11 is fixed. Could you please fix it?

Reply:
See also https://github.com/masutaka/compare_linker/blob/v1.4.3/lib/compare_linker/github_link_finder.rb#L19-L29

Reply:
OK, I make PR for https://github.com/chef/mixlib-cli.
Thanks!

Reply:
I made a PR to chef/mixlib-cli and merged it.
https://github.com/chef/mixlib-cli/pull/64

It will be released as v2.0.9.

![image](https://user-images.githubusercontent.com/10208211/59007929-94bc0280-8863-11e9-8d2d-8fd78acb9767.png)